source/_patterns/templates/typography/typography.hbs

```diff
@@ -0,0 +1,115 @@
+<div class="container-fluid">
+    <div class="pf-verticalRhythm">
```

Comment:
We don't use came classes for class names:
https://github.com/patternfly/patternfly-css/blob/master/CODE-GUIDELINES.md#naming-selectors

Can you change it to `pf-vertical-rhythm` please?